### ISSUE TITLE
Problem in SQLTrackerStore

### ISSUE THREAD

**Issue by alessandrarequena** [RasaHQ/rasa#4343]
**Rasa version**: 1.2.2

**Rasa X version** (if used & relevant): not used

**Python version**: 3.6

**Operating system** (windows, osx, ...): Windows 10 Pro

**Issue**:
When using SQLTrackerStore, the actions predictions seens crazy. I think it is because that when the events are retrieved from the database, no ordering is imposed. So, depending on the RDBMS used, the actions that are selected by the slice_state_history of MaxHistoryTrackerFeaturizer are random. 

So, I changed line 450 in the class SQLTrackerStore (python file tracker_store.py) to:

result = query.filter_by(sender_id=sender_id).order_by(self.SQLEvent.timestamp.asc()).all()

I just added the ordering clause (order by timestamp). As the slicer gets the end of the states array, I used ascending order.

Now the predictions seens ok.

**Error (including full traceback)**:
```
There is no traceback, the problem is an odd behavior from Rasa Core.
```

**Command or request that led to error**:
```
python -m rasa shell --debug
```

**Content of configuration file (config.yml)** (if relevant):
```yml
language: pt
pipeline:
  - name: 'components.ner_tcu.NerT'
  - name: 'components.entity_tagger.EntityTagger'
  - name: 'components.normaliza.Normaliza'
  - name: 'WhitespaceTokenizer'
  - name: 'CRFEntityExtractor'
  - name: 'EntitySynonymMapper'
  - name: 'CountVectorsFeaturizer'
  - name: 'EmbeddingIntentClassifier'

policies:
  - name: KerasPolicy
    epochs: 30
    batch_size: 8
    random_seed: 42
    featurizer:
      - name: MaxHistoryTrackerFeaturizer
        max_history: 3
        state_featurizer:
          - name: LabelTokenizerSingleStateFeaturizer
  - name: MappingPolicy
  - name: FallbackPolicy
    nlu_threshold: 0.7
    core_threshold: 0.6
    fallback_action_name: 'action_default_fallback'
```

**Content of domain file (domain.yml)** (if relevant):
```yml
It 's not relevant for the problem.
```

**Comment by sara-tagger** [comment]
Thanks for the issue, @amn41 will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by Bots4You-Heimel** [comment]
This is fixed in 1.3.0 but not in 1.2.x. Please apply the fix also to the pip releases, as 1.3.0x doesn't work for us yet.

**Comment by tmbo** [comment]
I think @ricwo backported that fix to 1.2.x - did you also create a release?

**Comment by ricwo** [comment]
yes, this was released as 1.2.8
